tests/pytest.ini

```diff
@@ -11,6 +11,8 @@ markers:
     sanity_check: override the default sanity check settings
     topology: specify which topology testcase can be executed on: (t0, t1, ptf, etc)
     platform: specify which platform testcase can be executed on: (physical, virtual, broadcom, mellanox, etc)
+    completeness_level: specify test completeness (coverage) level (Debug, Basic, Confident, Thorough)
```

Comment:
`completeness_level` is a CLI arg and also is the name for a marker that we use to carry the value of normalized level.
The marker's description is provided at:
https://github.com/Azure/sonic-mgmt/pull/1927#discussion_r458481818

To avoid the confusion, should I change the name of CLI arg to `specified_completeness_level`. Alternatively, the markers name to `normalized_completeness_level`?

Reply:
I would prefer to name the normalized marker as 'normalized_completeness_level'

Reply:
Or is there any harm in rewriting the current completeness_level markers specified in the testcase with the normalized level?

Reply:
Update - modified the code to use just one marker instead of two. The normalized completeness level is now added to the list of supported levels.